README.md

```diff
@@ -1,20 +1,30 @@
 # UoYWeek
-UoYWeek gives the current date string formatted as used by the [University of York](https://www.york.ac.uk/). For example, `spr/3/wed` would refer to Wednesday on the 3rd week of Spring term.
+
+UoYWeek gives the current date string formatted as used by the [University of
+York](https://www.york.ac.uk/). For example, `spr/3/wed` would refer to
```

Comment:
Leaving aside the issue of wrapping at 80chars, i wouldn't do it in the middle of a link